source/usb/usb_lib.c

```diff
@@ -1065,6 +1065,8 @@ void USBD_Reset_Event(void)
 #endif
 #endif  /* ((USBD_CDC_ACM_ENABLE)) */
 
+__weak void board_usb_sof_event(void) {}
```

Comment:
That should be __WEAK now

Reply:
I just tested this and the build fails when changing this to __WEAK without including "cmsis_compiler.h". Should I leave it as __weak instead in this PR?

Reply:
I am sorry, this will only have to be done after #701 is merged.